changehc/delphi_changehc/load_data.py

```diff
@@ -0,0 +1,137 @@
+"""
+Load CHC data.
+
+Author: Aaron Rumack
+Created: 2020-10-14
+"""
+
+# third party
+import pandas as pd
+
+# first party
+from .config import Config
+
+
+def load_denom_data(denom_filepath, dropdate, base_geo):
+    """Load in and set up denominator data.
+
+    Args:
+        denom_filepath: path to the aggregated denominator data
+        dropdate: data drop date (datetime object)
+        base_geo: base geographic unit before aggregation ('fips')
+
+    Returns:
+        cleaned denominator dataframe
+    """
+    assert base_geo == "fips", "base unit must be 'fips'"
```

Comment:
if this is true, why is it an argument and not a constant?

Reply:
We get other data available at both the fips level and hrr level, so I wanted to keep it flexible in case this data also becomes available at other geographic levels. I can change it though, whatever you think is best